File: LeetCode/Python/0485. Max Consecutive Ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        _max = 0
        count = 0
        
        for num in nums:
            if num:
                count += 1
                _max = max(_max, count)
            else:
                count = 0
        
        return _max

File: LeetCode/Java/0485. Max Consecutive Ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int count = 0;
        int max = 0;
        
        for (int num: nums) {
            if (num == 1) 
                max = Math.max(max, ++count);
            else
                count = 0;
        }
        
        return max;
    }
}
